server/lib/awsS3.ts

```diff
@@ -39,6 +40,11 @@ if (config.aws.s3.key) {
   });
 }
 
+export function getSignedGetURL(params: GetObjectCommand['input'], options?: { expiresIn: number }) {
+  const command = new GetObjectCommand(params);
+  return getSignedUrl(s3, command, { expiresIn: options?.expiresIn || 10 * 60 });
```

Comment:
:ice_cream: I'd put the `10 * 30` as a default parameter in the function prototype, to make it obvious for people using it.